
state_space:
  x_lower: -1000.0
  x_upper: 1000.0
  y_lower: -1000.0
  y_upper: 1000.0
  turning_radius: 10.0
  
planner:
  nav_msgs_path_topic: ompl_rs_planner_ros/nav_msgs_path
  planning_service_name: ompl_rs_planner_ros/planning_service
  path_frame: map
  path_spatial_resolution: 0.05
  planner_range: 15.0
  max_planning_time: 1.0
  